docs/messaging/rocketmq.md

```diff
@@ -19,7 +19,7 @@ Specific attributes for Apache RocketMQ are defined below.
 <!-- semconv messaging.rocketmq(full,tag=tech-specific-rocketmq) -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
-| [`messaging.rocketmq.client_group`](../attributes-registry/messaging.md) | string | Name of the RocketMQ producer/consumer group that is handling the message. The client type is identified by the SpanKind. | `myConsumerGroup` | Required |
```

Comment:
Producer groups were deprecated - https://rocketmq.apache.org/docs/domainModel/04producer/#version-compatibility

Also, the `client_group` was [never set by RocketMQ java instrumentation](https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-java-instrumentation+MESSAGING_ROCKETMQ_CLIENT_GROUP&type=code)